proposals/NNNN-Avoid-Deadlock.md

```diff
@@ -0,0 +1,31 @@
+# Add a Mechanism to Avoid Deadlock
+
+* Proposal: [SDL-NNNN](NNNN-Avoid-Deadlock.md)
+* Author: [Yuki Shoda](https://github.com/Yuki-Shoda)
+* Status: **Awaiting review**
+* Impacted Platforms: [iOS]
+
+## Introduction
+This proposal is to avoid deadlock by adding a mechanism that stops Audio Streaming under certain conditions when the app on Handset(HS) side moves to background.
```

Comment:
```suggestion
This proposal is to avoid deadlock by adding a mechanism that stops Audio Streaming under certain conditions when the app on Handset (HS) side moves to background.
```